### ISSUE TITLE
Alphafold all genes not already Alphafolded

### ISSUE THREAD

**Issue by iqbal-lab** [AllTheBacteria/AllTheBacteria#7]
Might as well, while we're about it. Depends on https://github.com/iqbal-lab-org/AllTheBacteria/issues/1  from @johnlees

Have started discussions within EBI about how to do this, and integrate the output with pdb etc

**Comment by SilasK** [comment]
You might want to consider the ESM algorithm. I have heard it's faster.

**Comment by iqbal-lab** [comment]
Yes, am considering it. But this is all only worthwhile for proteins that have not already been alphafolded , so current plan is to first see how many genes we have which are "novel" , and maybe prioritise
